api/v1/model/src/commonMain/kotlin/Repository.kt

```diff
@@ -33,7 +39,16 @@ data class Repository(
 
     /** The url to the repository. */
     val url: String
-)
+) {
+    companion object {
+        const val INVALID_URL_MESSAGE = "The entity url is malformed."
+        const val USER_INFO_MESSAGE = "The entity url must not contain userinfo."
```

Comment:
Since the messages are specific to repositories, they could run "The repository url..." instead of "entity url". And should URL be written in uppercase?